content/agent/basic_agent_usage/amazonlinux.md

```diff
@@ -2,86 +2,152 @@
+|:-----|:----|
+|`/etc/dd-agent/conf.d/`|`/etc/datadog-agent/conf.d/`|
+
+## Troubleshooting
+
+Run the info or status command to see the state of the Agent.
+The Agent logs are located in the `/var/log/datadog/` directory:
+
+* For Agent v6 all logs are in the `agent.log` file
+* For Agent v5 logs are in:
+    
+    * `datadog-supervisord.log`
+    * `collector.log`
+    * `dogstatsd.log`
+    * `forwarder.log`
+
+If you're still having trouble, [our support team](/help) will be glad to provide further assistance.
+
+## Adding a custom python package to the agent
+
+The Python interpreter embedded with the Agent is located here:  
+`/opt/datadog-agent/embedded/bin/python`.
+The agent also comes with pip, install python libraries using:
 ```
-To receive information about the Agent’s state:
-```shell
-sudo /etc/init.d/datadog-agent info
+sudo -u dd-agent -- /opt/datadog-agent/embedded/bin/pip install <package_name>
 ```
 
-Tracebacks for errors can be retrieved by setting the `-v` flag: (since 3.8.0)
-```shell
-sudo /etc/init.d/datadog-agent info -v
-```
+## Switch between Agent v5 and v6
+### Upgrade to Agent 6
+
+A script is available to automatically install or upgrade the new Agent. It sets up the repos and install the package for you; in case of upgrade, the import tool also searches for an existing `datadog.conf` from a prior version and converts Agent and checks configurations according to the new file format and filesystem layout.
+#### One-step install
+##### To Upgrade
+
+In case you have an Agent version 5.17 or later and you want to import the
+existing configuration:
 
-More information about the metrics, events, and service checks for an [integrations](/integrations) can be retrieved with the check command:
 ```shell
-sudo /etc/init.d/datadog-agent check [integration]
+ DD_UPGRADE=true bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
 ```
 
-Add the check_rate argument to get the most recent values for rates:
+**Note:** the import process won't automatically move custom checks, this is by
+design since we cannot guarantee full backwards compatibility out of the box.
+
+##### To Install Fresh
+
+To install on a clean box (or have an existing agent 5 install from which you do not wish to import the configuration) provide an api key:
+
 ```shell
-sudo /etc/init.d/datadog-agent check [integration] check_rate
+ DD_API_KEY=YOUR_API_KEY bash -c "$(curl -L https://raw.githubusercontent.com/DataDog/datadog-agent/master/cmd/agent/install_script.sh)"
 ```
 
-## Configuration
-The configuration file for the Agent is located at `/etc/dd-agent/datadog.conf`
+#### Manual install
+1. Set up apt so it can download through https
 
-Configuration files for integrations are located in `/etc/dd-agent/conf.d/
+    ```shell
+    sudo apt-get update
+    sudo apt-get install apt-transport-https
+    ```
 
-## Troubleshooting
+2. Add the beta repo to your system and import the datadog gpg key
```

Comment:
If we're merging this for the 6.0 release next week, it'll no longer *beta*

Reply:
Datadog not datadog?

Reply:
👌